elytron/src/main/java/org/wildfly/extension/elytron/SSLDefinitions.java

```diff
@@ -210,6 +210,14 @@
             .setDefaultValue(new ModelNode("DEFAULT"))
             .build();
 
+    static final SimpleAttributeDefinition CIPHER_SUITE_NAMES = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.CIPHER_SUITE_NAMES, ModelType.STRING, true)
+            .setAllowExpression(true)
+            .setMinSize(1)
+            .setRestartAllServices()
+            .setValidator(new CipherSuiteNamesValidator())
+            .setDefaultValue(new ModelNode("TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256"))
```

Comment:
Could this default value be a constant (afaict, it's the same value than in ServerSSLContext above)?

Reply:
is there any relation between `CIPHER_SUITE_NAMES` and `CIPHER_SUITE_FILTER`?

In their description, you wrote that:

```
elytron.server-ssl-context.cipher-suite-filter=The filter to apply to specify the enabled cipher suites for TLSv1.2 and below.
elytron.server-ssl-context.cipher-suite-names=The filter to apply to specify the enabled cipher suites for TLSv1.3.
```

I suppose that means that I can set both attributes and depending on the TLS version, one or the other would be used. Is that correct?

Reply:
> Could this default value be a constant (afaict, it's the same value than in ServerSSLContext above)?

I've updated both ServerSSLContext and SSLDefinitions to make use of a constant.

Reply:
> is there any relation between `CIPHER_SUITE_NAMES` and `CIPHER_SUITE_FILTER`?
> 
> In their description, you wrote that:
> 
> ```
> elytron.server-ssl-context.cipher-suite-filter=The filter to apply to specify the enabled cipher suites for TLSv1.2 and below.
> elytron.server-ssl-context.cipher-suite-names=The filter to apply to specify the enabled cipher suites for TLSv1.3.
> ```
> 
> I suppose that means that I can set both attributes and depending on the TLS version, one or the other would be used. Is that correct?

Yes, that's correct, both attributes can be set. If TLS 1.3 is supported, then the cipher suite will be selected using `CIPHER_SUITE_NAMES`. If TLS 1.3 is not supported, then the cipher suite will be selected using `CIPHER_SUITE_FILTER`. (There were many changes made to the way cipher suites work starting in TLS 1.3. As a result, pre-TLS 1.3 cipher suites cannot be used with TLS 1.3 and TLS 1.3 cipher suites cannot be used with TLS 1.2 or earlier.)